Dockerfile

```diff
@@ -11,10 +12,21 @@ ENV NPM_CONFIG_LOGLEVEL ${LOGLEVEL}
 # Install base dependencies
 RUN npm -g install npm@8.x.x
 
-WORKDIR /morello-api
+COPY package*.json ./
+COPY tsconfig.json ./
 
+RUN npm install
 COPY . .
+RUN npm run build
+
+# Service
+FROM node:16-alpine as service
+
+WORKDIR /morello-api
+COPY package*.json ./
```

Comment:
`RUN npm -g install npm@8.x.x`

Reply:
it has been added, thanks tested image locally without this and it has started the service successfully, I guess because we do not use npm to start it e.g. `CMD [ "node", "./index.js" ]` rather than `CMD [ "npm", "start" ]
`

Reply:
You need to install the latest version of NPM here as well as in the builder image otherwise you will end up with different packages installed.

Reply:
marking as resolved as this has been added to the service stage as well